src/index/index.tsx

```diff
@@ -30,11 +31,28 @@ const Home = ({
             <a title="Prosjektinfo">Les mer</a>
           </Link> */}
         </div>
-        <figure>
-          <div className={style.cases__decorationBox} />
-          <div className={style.cases__decorationBox} />
+        <DecorativeBoxes
+          box1Properties={{
+            color:
+              index % 3 === 0
+                ? '--color-secondary3'
+                : index % 3 === 1
+                ? '--color-secondary2__tint4'
+                : '--color-primary__tint3',
+            position: index % 2 === 1 ? 'top-right' : 'top-leftish',
```

Comment:
Dette ville jeg ha samlet opp enten som en funksjon eller en variabel et sted. Blir fort vanskelig å lese med nested terneries

Reply:
Har skrevet dette om til å heller bruke en variabel som ligger i en util-fil. Skrev også om typen jeg bruker for farge, nå bruker den heller typen som eksisterer fra før (AllColorNames, tror det var du som lagde den). Mye bedre som dette, tror jeg.

Reply:
Føler position er helt okei, siden det bare er en vanlig ternary som flipper frem og tilbake på par- og oddetall.